conformance-test.sh

```diff
@@ -6,36 +6,33 @@ venv() {
 	# shellcheck source=/dev/null
         source "$1"/bin/activate
 }
+
 version=${version:-v1.0}
+spec_branch=${spec_branch:-main}
+
 if [[ "$version" = "v1.0" ]] ; then
-   wget https://github.com/common-workflow-language/common-workflow-language/archive/main.tar.gz
-   tar xzf main.tar.gz && rm main.tar.gz
+    repo=common-workflow-language
 else
     # shellcheck disable=SC2001
-    repo=$(echo "$version" | sed 's/\(v[0-9]*\.\)\([0-9]*\).*/\1\2/')  
-    wget https://github.com/common-workflow-language/cwl-"${repo}"/archive/main.tar.gz
-    tar xzf main.tar.gz && rm main.tar.gz
+    repo=cwl-$(echo "$version" | sed 's/\(v[0-9]*\.\)\([0-9]*\).*/\1\2/')
 fi
+wget "https://github.com/common-workflow-language/${repo}/archive/${spec_branch}.tar.gz"
+tar xzf "${spec_branch}.tar.gz" && rm "${spec_branch}.tar.gz"
 
 docker pull node:slim
 
 # shellcheck disable=SC2043
-for CONTAINER in docker
-# for CONTAINER in docker singularity
-# singularity having issues on ci.commonwl.org; tests pass with https://gist.github.com/mr-c/0ec90d717617d074017c0cb38b72d1a4
-do
+for CONTAINER in docker singularity; do
 	venv cwltool-venv3
 	pip3 install -U setuptools wheel pip
 	pip3 uninstall -y cwltool
 	pip3 install -e .
-	pip3 install "cwltest>=1.0.20180518074130" codecov
-	if [[ "$version" = "v1.0" ]]
-	then
-		DRAFT="DRAFT=v1.0"
-		pushd common-workflow-language-main || exit 1
-	else
-		pushd cwl-"${repo}"-main || exit 1
+	pip3 install codecov
+	pip3 install git+https://github.com/common-workflow-language/cwltest.git@main#cwltest
```

Comment:
`cwltest` version 2.1 is out with your changes now :-) https://pypi.org/project/cwltest/2.1.20210423130452/